src/UnitTest/tests/Base/TestData.h

```diff
@@ -1291,10 +1291,92 @@ void Data_object::test< 25 >() {
          std::vector< tk::real >{ 0.1, 0.1, 0.1 }, e2.extract( 0, 1 ) );
 }
 
-//! Test tk::Data's operator/=
+//! Test tk::Data's operator*= by tk::real as rhs
 //! \author J. Bakosi
 template<> template<>
 void Data_object::test< 26 >() {
```

Comment:
Which magic numbers do you mean? The template arguments, e.g., 26? Or, e.g., the constructor arguments in `tk::Data`, and `fill()` inside the test function?